QuickOSM/quick_osm.py

```diff
@@ -159,6 +160,18 @@ def open_dialog(self):
         """Create and open the main dialog."""
         from QuickOSM.ui.dialog import Dialog
         dialog = Dialog()
+
+        if not get_setting("copyright_dialog"):
+            set_setting("copyright_dialog", "OpenStreetMap")
+            message = QMessageBox(dialog)
+            text = (
+                'Please, understand that you will be working with OpenStreetMap data.'
+                ' Therefore you must copyrigth your job with "OpenStreetMap" to be'
+                ' compliant with https://osm.org/copyrights.'
```

Comment:
https://www.openstreetmap.org/copyright

Can we click on the link ? does it work ?

"OpenStreetMap® is open data, licensed under the Open Data Commons Open Database License (ODbL) by the OpenStreetMap Foundation.

The Foundation requires that you use the credit “© OpenStreetMap contributors” on any product using OSM data."

Use a Yes/No QMessageBox.
If no, quit the dialog.

Reply:
Also use "tr" to translate